cmd/actions.go

```diff
@@ -15,9 +15,8 @@ import (
 var (
 	// CmdActions represents the available actions sub-commands.
 	CmdActions = &cli.Command{
-		Name:        "actions",
-		Usage:       "",
-		Description: "Commands for managing Gitea Actions",
+		Name:  "actions",
+		Usage: "Manage Gitea Actions",
```

Comment:
Go formatting is so weird, why does this suddenly de-indent? :D

Reply:
```
Short1:      1,
Short2:      2,
Loooooooong: 3,
```

Then

```
Short1: 1,
Short2: 2,
```

Reply:
Yes, but the `Subcommands` should cause indentation but doesn't.

Reply:
Yup, IIRC complex values exempt from "indentation rules"